ipaserver/plugins/dogtag.py

```diff
@@ -2019,8 +2019,11 @@ def get_client(self):
             str(self.kra_port),
             'kra')
 
-        connection.set_authentication_cert(paths.RA_AGENT_PEM,
-                                           paths.RA_AGENT_KEY)
+        connection.session.cert = (paths.RA_AGENT_PEM, paths.RA_AGENT_KEY)
+        # uncomment the following when this commit makes it to release
+        # https://git.fedorahosted.org/cgit/pki.git/commit/?id=71ae20c
+        # connection.set_authentication_cert(paths.RA_AGENT_PEM,
+        #                                    paths.RA_AGENT_KEY)
```

Comment:
This is a hack to the commented solution, pki-base.10.3.5-11 available on Fedora still does not provide the method I modified in PKI. We'd better use the public methods of objects we work with instead of injecting their attributes in case their implementation changes.